Homework_4/questions.md

```diff
@@ -0,0 +1,22 @@
+### Question 1
+Complete definitions for key Git & GitHub terminology
+
+GIT WORKFLOW FUNDAMENTALS
+
+* Working Directory - it is a current directory where we want to work on it with our files. 
+* Staging Area - it is a space with drafts, files waiting till next commitment.
+* Local Repo (head) - for each branch is a one head  = it's a current/working branch
+* Remote repo (master) = default branch 
+
+WORKING DIRECTORY STATES:
+* Staged - file can be added to the local database
+* Modified - file were changed after commitment
+* Committed - files are stored in local database
+
+GIT COMMANDS:
+* Git add - changes from the working directory to the staging are
+* Git commit - takes staged files and commits to the project database
+* Git push - it lets to move a local branch to another repository
```

Comment:
ℹ️ the word another repository is not quite right, it should be a remote repository.